### ISSUE TITLE
Bug with system default embargo "None"

### ISSUE THREAD

**Issue by StephanieVUGSC** [TexasDigitalLibrary/Vireo#1482]
If the system default embargo for None has the box UNchecked for "This embargo option will be available for new submissions" and a custom None (0 months) embargo has been created to display in new submissions, the System default  will appear in the student submission view **after** the student selects the local None embargo.  Both System default  and local "None" embargoes displays but the system default gets the radio button selection.  

Unable to reproduce bug on embargoes of 12 months, 24 months, or indefinite.

**Comment by mtpatterson** [comment]
Seems to be related to the actual name, when you change the custom "None" to a different name and then go back to the submission page with embargo selection and check a new radio button the default "None" disappears.

Also worth noting is that the default "None" isn't even in the submission page's source code in the browser until you select the custom "None".

When you check the radio button the page makes a network call to http://localhost:9000/submission/3/update-field-value/73 and then updates all possible Embargo selections.

Looking more into this later and hopefully resolving the issue, but this is what I've found.

**Comment by mtpatterson** [comment]
Default behavior is to not allow multiple custom embargoes of the same name at all, is it intended behavior to be able to make a custom embargo with the same name as a default embargo?

**Comment by StephanieVUGSC** [comment]
Not exactly, but since we are unable to edit the system embargo descriptions the additional limitation on names is not desireable.  Vireo 3 does not have any problems with duplicate names.   

Perhaps this can be taken care of by noting the limitation in the documentation and suggesting use of "No embargo" instead of "None".

**Comment by cstarcher** [comment]
Either fix by overriding the delete method or fix in weaver.

**Comment by cstarcher** [comment]
This is still a bug in 4.2.7.
